packages/wallet-api-manifest-validator/README.md

```diff
@@ -0,0 +1,120 @@
+  </p>
+</p>
+
+## Quick intro
+
+The manifest Validator is a typescript package that checks if your `manifest.json` file meets the requirements for Ledger Live App manifest submission.
+
+To achieve this, the package uses [JSON Schema](https://json-schema.org/), a vocabulary for annotating and validating JSON documents.
+
+In top of that, we us the [AJV](https://ajv.js.org/) and [AJV-errors](https://www.npmjs.com/package/ajv-errors) libraries which provides more possibilities (please refer to [AJV Github](https://github.com/ajv-validator/ajv) to know which version of JSON Schema we use).
+
+## Installation
+
+```bash
+npm i @ledgerhq/wallet-api-manifest-validator
+
+```
+
+_Can also be used globaly :_
+
+```bash
+npm i -g @ledgerhq/wallet-api-manifest-validator
+```
+
+## How to use
+
+#### Import
+
+```typescript
+import { validateManifest } from "@ledgerhq/wallet-api-manifest-validator";
+import MyjsonManifestFile from "./manifest.json";
+
+const result: boolean = validateManifest(MyjsonManifestFile);
+```
+
+#### CLI
+
+Once you have installed it, use the command :
+
+```bash
+npm run validate <*fileOrDirectoryName*>
+```
+
+_Or if you have installed it in global :_
+
+```bash
+validate <*fileOrDirectoryName*>
+```
+
+help command
+
+```bash
+npm run validate -h
+```
+
+#### More info about the validator options
+
+The validateManifest function takes 1 mandatory argument and options.
+
+**Arguments**
+
+|   Args   |  Type  | Description                                                   |
+| :------: | :----: | ------------------------------------------------------------- |
+| manifest |  JSON  | Your JSON file. <span style="color:red">NOT your path</span>. |
+| options  | Object | More details bellow                                           |
+
+**Options**
+
+|   Options   |  Type   | Description                                                                                 |
+| :---------: | :-----: | ------------------------------------------------------------------------------------------- |
+|   details   | boolean | describe errors when they occur                                                             |
+| enableState | boolean | result description (e.g. show in console "The JSON file does not correspond to the schema") |
+|  fileName   | string  | file name, practical to validate multiple files via multiple calls                          |
+
+Imported in source code:
+
+```typescript
+validateManifest(MyjsonManifestFile, { details, enableState, fileName });
+```
+
+_Cli command don't allow you to use the fileName option._
+_--throwError option is only available on cli._
+
+Using CLI:
+
+```bash
+npm run validate *fileOrDirectoryName* --details --enableState --throwError
+```                                                                                                                                          
+
+</br>
+</br>
+</br>
+
+more infos about the manifest here :
+
+https://developers.ledger.com/docs/dapp/manifest/
```

Comment:
You should also link to the non dapp manifest page -> https://developers.ledger.com/docs/non-dapp/tutorial/2-manifest/

At least until to doc is reworked and we have only one page to document manifests (probably directly in this repo in the Docusaurus doc once it will be setup)

```suggestion
- https://developers.ledger.com/docs/dapp/manifest/
- https://developers.ledger.com/docs/non-dapp/tutorial/2-manifest/
```